### PR TITLE
Launch Leaders

### PR DESCRIPTION
**Pull request by lachlanjc**

### CODE REVIEW

File: src/components/home/Header.js [hunk 1]
```diff
@@ -123,26 +123,26 @@ export default () => (
...
-        <A.link
+        <A
           caps
           color="info"
           fontSize={2}
           px={2}
           bold
           ml="auto"
-          to="/start"
+          to="https://leaders.hackclub.com/?ref=home"
```
Comment (zanedb) [inline src/components/home/Header.js:141]: Shouldn't this be `href="https://leaders.hackclub.com/?ref=home"`?

**Review by zanedb — CHANGES_REQUESTED** [review]

File: src/components/home/Header.js (hunk 1 above)
Comment (lachlanjc) [reply src/components/home/Header.js:141]: Yesss thx. We were kinda done when we wrote this, lol

Comment (zanedb) [reply src/components/home/Header.js:141]: I understand :)

**Comment by lachlanjc** [comment]
@MaxWofford 👋

**Comment by maxwofford** [comment]
LGTM

**Comment by lachlanjc** [comment]
Max, this doesn’t work, it needed to be fixed before merging :)

**Comment by maxwofford** [comment]
Published the previous build on Netlify

**Comment by maxwofford** [comment]
@lachlanjc will let you handle making a fix, but I like the design. Unclear on why I was originally tagged on this though

**Pull request merged**